test/pass_files/throwattribute.d

```diff
@@ -0,0 +1,10 @@
+void doStuff() throw {
+	throw void doThings()
```

Comment:
What about a fail case for `void foo (throw int x)`? Or does it not matter / it's already handled?

Reply:
It doesn't parse.

Reply:
And, actually, putting a test case for that into the fail_files directory was really easy, so I did it.